tfe.go

```diff
@@ -751,7 +775,16 @@ func checkResponseCode(r *http.Response) error {
 		}
 	}
 
-	return fmt.Errorf(strings.Join(errs, "\n"))
+	return errs, nil
+}
+
+func errorPayloadHasMatch(errors []string, match string) bool {
```

Comment:
💅 maybe call it errorPayloadContains because it better reflects that it'll use string.Contains